src/helpers/jwks.js

```diff
@@ -21,8 +21,11 @@ function checkStatus(response) {
   return Promise.reject(error);
 }
 
-export function getJWKS(options, cb) {
-  const localFetch = typeof fetch == 'undefined' ? unfetch : fetch;
+function getFetchInstance() {
+  return typeof fetch == 'undefined' ? unfetch : fetch;
+}
+
+export function getJWKS(options, cb, localFetch = getFetchInstance()) {
```

Comment:
In Node the convention is callback is always the last argument, also I'd rather not change `getJWKS` - you can stub global fetch with sinon rather than changing the signature of this